app/views/_components/feedback/_feedback.scss

```diff
@@ -1,6 +1,6 @@
 .app-feedback__section {
   position: relative;
-  padding-right: 70px;
+  // padding-right: 70px;
```

Comment:
Delete rather than comment out?

Reply:
Oh I should be cleaning this up completely. Thanks.